Makefile

```
@@ -17,8 +17,8 @@ develop:
 dockerhub-update-multiarch:
 	@echo "If the container builds successfully, do this to push to dockerhub:"
 	@echo "Run 'docker login'"
-	@#if no builder exists yet:
-	docker buildx create --name container --driver=docker-container
+	@#if no builder exists yet, create one
+	docker buildx inspect || docker buildx create --name container --driver=docker-container
```

Did you meant to do
```suggestion
	docker buildx inspect container|| docker buildx create --name container --driver=docker-container
```

Otherwise could you suggest how the `docker build inspect` is useful here? ( I assuem there is always a default builder)